tendermint/src/lite_impl/signed_header.rs

```diff
@@ -52,42 +45,109 @@ impl lite::Commit for block::signed_header::SignedHeader {
     }
 
     fn validate(&self, vals: &Self::ValidatorSet) -> Result<(), Error> {
-        if self.commit.precommits.len() != vals.validators().len() {
+        // TODO: self.commit.block_id cannot be zero in the same way as in go
```

Comment:
only the very first block (height: 1) is allowed (actually required) to have 0 signatures